File: Data Structures & Algorithms/merge-triplets-to-form-target/submission-0.cpp
class Solution {
public:
    bool mergeTriplets(vector<vector<int>>& triplets, vector<int>& target) {

        bool res[3] = {false, false, false};
        
        for (auto& t : triplets) {
            if (t[0] > target[0] || t[1] > target[1] || t[2] > target[2]) {
                continue;
            }
            
            for (int i = 0; i < 3; i++) {
                if (t[i] == target[i]) {
                    res[i] = true;
                }
            }
            
            
            if (res[0] && res[1] && res[2]) return true;
        }
        
        return res[0] && res[1] && res[2];
    }
};

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        left = 0
        n = len(numbers)
        right = n-1

        while left < right:
            crsm = numbers[left] + numbers[right]

            if crsm == target:
                return [left+1, right+1]

            elif crsm > target:
                right -=1
            
            else:
                left +=1
        